exporters/otlp/internal/otlpconfig/tls.go

```diff
@@ -21,6 +21,7 @@ import (
 	"io/ioutil"
 )
 
+//nolint:revive
```

Comment:
I would like to know what was ignored and why
```suggestion
//nolint:revive // ignoring missing comments for unexported constants in an internal package
```